tools/fleetctl-npm/run.js

```diff
@@ -5,6 +5,7 @@ const { spawnSync } = require("child_process");
 const { existsSync, mkdirSync } = require("fs");
 const { type } = require("os");
 const { join } = require("path");
+const { arch } = require("process");
```

Comment:
`arch` on amd64 will be the `x64` string, right? (which won't match)

Reply:
Excellent point.  I'd argue we should fix this in the filenames rather than in the logic, because otherwise we'll have to repeat the `amd64` -> `x86` transformation everywhere we use it.  I'll look at the best way to to do that w/ Goreleaser.

Reply:
`amd64` is the value used by `GOARCH` on Go applications. 

`amd64` -> `x86`, did you mean `x64`?